code/game/gamemodes/autotraitor/autotraitor.dm

```diff
@@ -97,16 +97,18 @@
 
 	if(traitor_count < max_traitors)
 		if(prob(traitor_prob))
-			message_admins("Making a new Traitor.")
+			log_and_message_admins_no_usr("Making a new Traitor.")
 			if(!length(possible_traitors))
-				message_admins("No potential traitors. Cancelling new traitor.")
+				log_and_message_admins_no_usr("No potential traitors. Cancelling new traitor.")
 				addtimer(CALLBACK(src, PROC_REF(traitor_check_loop)), 15 MINUTES)
 				return
 			var/datum/mind/new_traitor_mind = pick(possible_traitors)
 			var/mob/living/new_traitor = new_traitor_mind.current
 
 			to_chat(new_traitor, "<span class='danger'>ATTENTION:</span> It is time to pay your debt to the Syndicate...")
 			new_traitor.mind.add_antag_datum(/datum/antagonist/traitor)
+			message_admins("[key_name(new_traitor)] was added in as a traitor!")
+			log_game("[new_traitor] was added in as a traitor.")
```

Comment:
Consider using `key_name()` in `log_game()` too, it helps logdiving if we can both search for ckey+charname (plus it is how it is done almost everywhere else in the code whenever we use that proc).

```suggestion
			log_game("[key_name(new_traitor)] was added in as a traitor.")
```

Reply:
Yeah I agree. I think I need to look at this again and see how I am logging even the other changes. I spaced on the fact that this will be handy when it comes to investigations post round.